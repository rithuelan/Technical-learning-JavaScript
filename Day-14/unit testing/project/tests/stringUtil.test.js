const { capitalize, reverse, capitalizeWords } = require('../src/stringUtil');

test('capitalizes first letter', () => {
  expect(capitalize('hello')).toBe('Hello');
});

test('reverses a string', () => {
  expect(reverse('abc')).toBe('cba');
});

test('capitalizes all words', () => {
  expect(capitalizeWords('hello world')).toBe('Hello World');
});

test('snapshot test for object', () => {
  const obj = { name: 'Alice', age: 25 };
  expect(obj).toMatchSnapshot();
});
