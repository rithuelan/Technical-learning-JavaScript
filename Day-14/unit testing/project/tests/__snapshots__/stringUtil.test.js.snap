// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for object 1`] = `
{
  "age": 25,
  "name": "Alice",
}
`;
